go/pkg/config/load.go

```diff
@@ -142,12 +152,23 @@ func Parse(text []byte, dir string) (conf *Config, err error) {
 func FindConfigPath(startFolder string) (configPath string, deprecatedRepositoryProjectRoot string, err error) {
 	folder := startFolder
 	for i := 0; i < maxSearchDepth; i++ {
-		configPath = filepath.Join(folder, global.ConfigFilename)
-		exists, err := files.FileExists(configPath)
+		// .yaml
+		configPath = filepath.Join(folder, global.ConfigFilename[0])
```

Comment:
Could this be turned into a for loop over the members of `global.ConfigFilename`?

Reply:
Done